### PR TITLE
Fix capacity update command when the capacity in Paused state

### PR DESCRIPTION
**Pull request by selasagi**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by selasagi** [comment]
Replace the PR with new one to make it cleaner

**Comment by selasagi** [comment]
Please review

File: src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -9,8 +9,8 @@
 [assembly: AssemblyTitle("Microsoft Azure PowerBI Dedicated Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for PowerBI Dedicated Resources.")]
 
-[assembly: AssemblyVersion("0.9.0.0")]
-[assembly: AssemblyFileVersion("0.9.0.0")]
+[assembly: AssemblyVersion("0.10.0.0")]
```
Comment (dsgouda) [inline src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Properties/AssemblyInfo.cs]: Please do not modify the `AssemblyVersion`

File: src/SDKs/_metadata/powerbidedicated_resource-manager.txt [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-Branch:      current
-Commit:      5e44f1a49008963f1b189dbd91903a22c470f757
+Branch:      master
+Commit:      1e2f0dae5c44d55b642d27232f9dcd7c0876488a
 
 2) AutoRest information
 Requested version: latest
-Bootstrapper version:    C:\Users\sagis\AppData\Roaming\npm `-- autorest@2.0.4166  
-Latest installed version:    2.0.4168
+Bootstrapper version:    C:\Program Files\nodejs `-- autorest@2.0.4283
```
Comment (dsgouda) [inline src/SDKs/_metadata/powerbidedicated_resource-manager.txt]: Please pull down latest psSdkJson6 changes and run `msbuild build.proj` and regenerate the code

File: src/SDKs/PowerBIDedicated/PowerBIDedicated.Tests/PowerBIDedicated.Tests.csproj [hunk 3]
```diff
@@ -2,7 +2,7 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />  
   <PropertyGroup>
     <PackageId>PowerBIDedicated.Tests</PackageId>
-    <VersionPrefix>0.9.0-preview</VersionPrefix>
```
Comment (dsgouda) [inline src/SDKs/PowerBIDedicated/PowerBIDedicated.Tests/PowerBIDedicated.Tests.csproj]: Test project version must always be set to `1.0.0`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Left a few comments

**Comment by dsgouda** [comment]
@selasagi please reference the REST spec PR

**Comment by selasagi** [comment]
Spec PR - https://github.com/Azure/azure-rest-api-specs/pull/1905

File: src/SDKs/PowerBIDedicated/PowerBIDedicated.Tests/PowerBIDedicated.Tests.csproj (hunk 3 above)
Comment (dsgouda) [reply src/SDKs/PowerBIDedicated/PowerBIDedicated.Tests/PowerBIDedicated.Tests.csproj]: Like pointed out earlier, this must be set to `1.0.0`, not `1.0.0-preview`
The `AssemblyInfo.cs` for the test project must also be set to `1.0.0`

File: src/SDKs/_metadata/powerbidedicated_resource-manager.txt (hunk 2 above)
Comment (dsgouda) [reply src/SDKs/_metadata/powerbidedicated_resource-manager.txt]: Still waiting on this

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 4]
```diff
@@ -142,4 +142,9 @@ build.out
 # Azure Project
 csx/
 *.GhostDoc.xml
-pingme.txt
\ No newline at end of file
+pingme.txt
```
Comment (dsgouda) [inline .gitignore]: Why was this added

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good, please undo changes made to the .gitignore.

File: .gitignore (hunk 4 above)
Comment (selasagi) [reply .gitignore]: Just miss it in VS ..
BTW this SDK will be used in the next Powershell release? Any idea when this should be?

**Review by dsgouda — APPROVED** [review]
LGTM Will merge on CIs passing

**Comment by dsgouda** [comment]
@selasagi  Please contact the PS team for their release schedule

**Comment by selasagi** [comment]
To publish I need first that my code will be merged. When you expect this merge?

**Comment by dsgouda** [comment]
@selasagi Please find instructions to publish [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/Documentation/code-generation-instructions.md#how-to-signpublish-bits)
